src/color/colors-brand.story.tsx

```diff
@@ -89,6 +91,48 @@ export function Taupe() {
 }
 Taupe.storyName = 'taupe';
 
+function ColorSwatch({ color, title, ...props }) {
+  const { red, green, blue } = parseToRgb(color);
+  const rbgString = `rgb(${red}, ${green}, ${blue})`;
+  return (
+    <div
+      style={{
+        display: 'inline-block',
+        overflow: 'hidden',
+        paddingRight: '1rem',
+      }}
+      {...props}
+    >
+      <ColorSwatchStyled color={color} />
+      <Header size="5">{title}</Header>
+      <Header size="5">{color}</Header>
+      <Header size="5">{rbgString}</Header>
+    </div>
+  );
+}
+
+const ColorSwatchStyled = styled.div`
+  width: calc(3rem + 1.25vw);
+  height: calc(3rem + 1.25vw);
+  background: ${({ color }) => color};
+  margin-right: 0.125rem;
+  border-radius: 0.25rem;
+`;
+export function Violet() {
+  const violetCache = [
```

Comment:
Since the violet color cache doesn't follow the other cache calculations, it will be explicitly referenced here.